app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3:
+        return
+
+    mv, filename, destination_path = parts
+    path, new_filename = os.path.split(destination_path)
+
+    if mv != "mv":
+        return
+
+    if not os.path.exists(filename):
+        return
```

Comment:
it is considered better because it enhances code readability, reduces nesting, and is more concise while still expressing the same conditions for the file move to occur